doc/api/util.md

```diff
@@ -3305,6 +3305,29 @@ util.isUndefined(null);
 // Returns: false
 ```
 
+### `util.loadEnvFile(path)`
```

Comment:
I feel like this should live on `node:process`. That’s where `process.env`, is, so then we could keep all environment variable stuff together. Unless adding something new to `process` incurs a performance penalty, because `process` is in the global scope?